jumpscale/packages/vdc_dashboard/chats/matic.py

```diff
@@ -55,12 +63,7 @@ def _enter_nodetype(self):
         else:
```

Comment:
is this block meant for handling `Sentry Node` or not needed?

Reply:
Yes, default is sentry